### PR TITLE
switching to dep

### PR DESCRIPTION
**Pull request by syossan27**
Please look at it.
https://github.com/Masterminds/glide/releases/tag/v0.13.0

### CODE REVIEW

**Comment by lestrrat** [comment]
If you are going to change all uses of glide in this project (please see the test failure in Travis, and the Makefile) I think it's worth considering. But otherwise I really don't see any immediate reason to switch to dep, as the glide binaries are still available, and there are no blockers caused by glide.

**Comment by syossan27** [comment]
Thanks for replying !
I'll fix .travis.yml and Makefile.
Please wait a little more.

**Comment by syossan27** [comment]
Fixed for TravisCI failed.
Please confirm.

File: Makefile [hunk 1]
```diff
@@ -37,45 +27,30 @@ $(INTERNAL_BIN_DIR):
...
-installdeps: glide $(SRC_FILES)
+installdeps: dep $(SRC_FILES)
 	@echo "Installing dependencies..."
-	@$(INTERNAL_BIN_DIR)/$(THIS_GOOS)/$(THIS_GOARCH)/glide install
+	@$(INTERNAL_BIN_DIR)/$(THIS_GOOS)/$(THIS_GOARCH)/dep ensure
 
 build-windows-amd64:
 	@$(MAKE) build GOOS=windows GOARCH=amd64 SUFFIX=.exe
 
-build-windows-386:
```
Comment (lestrrat) [inline Makefile]: Why is this deleted?

File: Makefile [hunk 2]
```diff
@@ -37,45 +27,30 @@ $(INTERNAL_BIN_DIR):
...
-	@$(MAKE) build GOOS=windows GOARCH=386 SUFFIX=.exe
-
 build-linux-amd64:
 	@$(MAKE) build GOOS=linux GOARCH=amd64
 
-build-linux-arm:
```
Comment (lestrrat) [inline Makefile]: Why is this deleted?

File: Makefile [hunk 3]
```diff
@@ -37,45 +27,30 @@ $(INTERNAL_BIN_DIR):
...
-	@$(MAKE) build GOOS=linux GOARCH=arm
-
-build-linux-arm64:
```
Comment (lestrrat) [inline Makefile]: Why is this deleted?

File: Makefile [hunk 4]
```diff
@@ -37,45 +27,30 @@ $(INTERNAL_BIN_DIR):
...
-	@$(MAKE) build GOOS=linux GOARCH=arm64
-
-build-linux-386:
```
Comment (lestrrat) [inline Makefile]: Why is this deleted?

File: Makefile [hunk 5]
```diff
@@ -37,45 +27,30 @@ $(INTERNAL_BIN_DIR):
...
-	@$(MAKE) build GOOS=linux GOARCH=386
-
 build-darwin-amd64:
 	@$(MAKE) build GOOS=darwin GOARCH=amd64
 
-build-darwin-386:
```
Comment (lestrrat) [inline Makefile]: Why is this deleted?

File: Makefile [hunk 6]
```diff
@@ -140,7 +103,8 @@ release-upload: release release-github-token
 
 test: installdeps
 	@echo "Running tests..."
-	@PATH=$(INTERNAL_BIN_DIR)/$(GOOS)/$(GOARCH):$(PATH) go test -v $(shell $(INTERNAL_BIN_DIR)/$(THIS_GOOS)/$(THIS_GOARCH)/glide nv)
+	# @PATH=$(INTERNAL_BIN_DIR)/$(GOOS)/$(GOARCH):$(PATH) go test -v $(shell $(INTERNAL_BIN_DIR)/$(THIS_GOOS)/$(THIS_GOARCH)/glide nv)
```
Comment (lestrrat) [inline Makefile]: Please delete this line instead of commeting it out.

**Comment by syossan27** [comment]
Thanks for confirm !

I deleted some of the build rules, because it was not included in the released build of dep.
I'd like to correspond, but dep v0.3.0 is supported by
- darwin-386
- darwin-amd64
- freebsd-386
- freebsd-amd64
- linux-386
- linux-amd64

dep v0.3.1 , 0.3.2 is supported by
- darwin-amd64
- linux-amd64
- windows-amd64

Since there was no build for 32bit in the latest release, I will try to use the build of v0.3.0.

However, since there is no released build of linux arm, arm64, I am worried about how to deal with it.
Once I thought about how to get the source code from dep, build it for arm, arm64, and use it, is there any suggestion?

**Comment by lestrrat** [comment]
Ahhhhh, then I don't think it's a good idea to migrate to dep for the time being. If you know about dep's future plans, please let me know.

For the record, we don't have to have full support for those build environments, but I don't want to gratuitously cut support for things that we had been supporting only because a support tool doesn't support them.

For the time being peco will stay with glide.

**Comment by syossan27** [comment]
I understand this matter.
Thank you for your support !
